src/pages/GeneralAdminView/GeneralAdminUserChange/GeneralAdminUserChange.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import languageSelector from '../../../assets/languages/languageSelector';
+import { toast } from 'react-toastify';
+import { AiFillDelete } from 'react-icons/ai';
+import axios from 'axios';
+import Cookies from 'universal-cookie';
+import './GeneralAdminUserChange.css'
+
+const GeneralAdminUserChange = () => {
+  const language = useSelector(state => state.languageReducer);
+  const [email, setEmail] = useState('');
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [users, setUsers] = useState([]);
+  const cookie = new Cookies();
+  const token = cookie.get('token');
+  const config = {
+    headers: {
+      'Authorization': `Bearer ${token}`
+    }
+  };
+
+
+  const fetchUsers = async () => {
+    try {
+      const response = await axios.get(`${process.env.REACT_APP_API_URL}/users`, {
+        params: { role: 'user' },
+        ...config,
+      });
+      setUsers(response.data.data);
+    } catch (error) {
+      toast.error(error.response?.data?.message || languageSelector(language, 'usersFetchError'));
+    }
+  };
+  
+  useEffect(() => {
+    fetchUsers();
+  }, []);
+  
+  const toggleExpanded = () => {
+    setIsExpanded(!isExpanded);
+  };
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const response = await axios.put(`${process.env.REACT_APP_API_URL}/users/change-role`, {
```

Comment:
I consider API_URL should be imported using a constant through the project. CC @dgaleano201611

Reply:
Resolved